sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJUnitMixinPlugin.scala

```diff
@@ -0,0 +1,30 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js sbt plugin        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+package org.scalajs.sbtplugin
+
+import sbt.Keys._
+import sbt._
+
+object ScalaJUnitMixinPlugin extends AutoPlugin {
+
+  override def projectSettings: Seq[Setting[_]] = Seq(
+    libraryDependencies ++= {
+      if (enablePlugin(scalaVersion.value)) {
+        Seq("org.scala-js" %
+            ("junit-mixin-support-scala-for-scala-2-12-_" + scalaVersion.value) %
+            "0.0.1-SNAPSHOT" % "plugin->test")
```

Comment:
It's good to know that it works with this manual hack, though, as we have some confidence that our entire test suite passes with M5 (as opposed to: bugs are hidden because their tests are not run).